src/søknad/barna-dine/LeggTilBarn.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
+import useSøknadContext from '../../context/SøknadContext';
+import { differenceInYears } from 'date-fns';
+import { RadioPanel } from 'nav-frontend-skjema';
+import { formatDate, formatDateFnr } from '../../utils/dato';
+import LeggTilBarnFødt from './LeggTilBarnFødt';
+import LeggTilBarnUfødt from './LeggTilBarnUfødt';
+import { IBarn } from '../../models/person';
+const uuidv4 = require('uuid/v4');
+
+interface Props {
+    settÅpenModal: Function;
+}
+
+const LeggTilBarn: React.FC<Props> = ( { settÅpenModal }) => {
+  const { søknad, settSøknad } = useSøknadContext();
+  const [barnDato, settBarnDato] = useState<Date>(new Date());
```

Comment:
Ser du har dratt inn momentjs, men bruker Date her. Er det bevist?

Reply:
Jeg har brukt date-fns til mine Date greier. Vurderte momentjs, men den skal være litt treigere enn date-fns. Tror nok at det er greit om vi bestemmer oss for det ene eller det andre. :) 

https://inventi.studio/en/blog/why-you-shouldnt-use-moment-js

Reply:
Ja, har noen hjelpefunksjoner i `dato.ts` som tar inn Date og bruker moment for å formatere.

Reply:
Jeg har valgt å bruke date-fns på mine Date ting. Vurderte å bruke momentjs i starten, men leste at den var ganske treig 🤷‍♀  Kanskje

https://inventi.studio/en/blog/why-you-shouldnt-use-moment-js

Reply:
Ah, ja, vi kan fjerne moment. Brukes så vidt uansett, så kan heller bruke date-fns der det trengs.